pages/Documents/ConversationalAI/Third-PartyBots/public-api.md

```diff
@@ -347,7 +356,7 @@ Example payload of the request with Simple Text, Pause/Delay, Private Text, [Str
 **Example cURL**:
 
 {: .important}
-**Please note** Make sure to replace [`{botDomain}`](#step-1-identify-the-third-party-bots-api-domain), `{accountId}`, `{conversationId}` and [`{bearerToken}`](#step-2-get-bearer-token) from the below command with your information
```

Comment:
I think it's unnecessary to. have the "please note" remark if it's already visualised with {: .important}
In general we should keep a consistent style for this throughout the third party bots docs.
What do you think? Check also the changes below

Reply:
Sure removed all please notes in the important part in this file